docs/conf.py

```diff
@@ -29,7 +29,7 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ["myst_parser"]
+extensions = ["myst_parser", "sphinxcontrib.mermaid"]
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
```

Comment:
Oh neat, I didn't know mermaid did themes!

I'll have a quick look - if it's not much more work, and sphinx supports it I'll try adding it, otherwise I'll make a separate issue, as the functionality of having diagrams was the key idea.

Reply:
hmm… this looks a bit more complicated than I thought, @roald-teunissen. I'd prefer to merge this in and pick up this work another time. The issue for it is below.

https://github.com/thegreenwebfoundation/admin-portal/issues/323